SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -117,9 +124,13 @@ - (void)sdl_applicationWillEnterForeground:(NSNotification *)notification {
 #pragma mark - Stream Lifecycle
 
 - (void)connect {
+    [self sdl_connect:nil];
+}
+
+- (void)sdl_connect:(EAAccessory *)accessory {
```

Comment:
Ah, I see it now. Could this be named `sdl_connectAccessory:`. It would be nice to have some API documentation for this method explaining what it does and what happens when `nil` is passed.